Doc/library/sqlite3.rst

```diff
@@ -472,6 +454,25 @@ Module constants
    This flag may be combined with :const:`PARSE_COLNAMES` using the ``|``
    (bitwise or) operator.
 
+   Generated fields (for example ``MAX(data)``) are returned as :class:`str`;
+   use :const:`!PARSE_COLNAMES` to enforce types for such queries.
+
+.. data:: PARSE_COLNAMES
+
+   Pass this flag value to the *detect_types* parameter of
+   :func:`connect` to look up a converter function by
+   using the type name, parsed from the query column name,
+   as the converter dictionary key.
+   The query column name must be wrapped with double quotes (``"``),
```

Comment:
"with" or "in"?

```suggestion
   The query column name must be wrapped in double quotes (``"``),
```

Reply:
How about "must be surrounded by double quotes"?

Reply:
I think I'll go for "in" for now (the following sentence uses "in").